OTRExporter/CutsceneExporter.cpp

```diff
@@ -450,8 +450,8 @@ void OTRExporter_Cutscene::SaveOot(ZCutscene* cs, BinaryWriter* writer) {
 }
 // MUST be uint16_t for -1
 typedef enum : uint16_t{
-    /* -1 */ CS_TEXT_TYPE_NONE = -1,
```

Comment:
I had the type of the enum by -1. Does GCC not follow it for `-1`? MSVC did.

Reply:
> I had the type of the enum by -1

I'm not I understand what you mean but -1 is not a valid value for uint16_t, which is why I made a cast which will turn it into `0xFFFF`. Maybe actually writing `0xFFFF` would be even better in this case, I lack some context as I don't really know how cutscene work. Otherwise, if -1 is really correct semantically then maybe this should be `int16_t` instead.